### PR TITLE
Use the same sentence for WD and CR

### PR DESCRIPTION
**Pull request by plehegar**
This change makes the stability sentence to be the same for WDs and CRDs.

### CODE REVIEW

**Review by deniak — APPROVED** [review]

**Comment by fantasai** [comment]
No objection, but would it make sense to use the shorter phrasing across all the statuses? (Maybe in the next revision if not in this one?)

**Comment by dontcallmedom** [comment]
+1 on using the shorter language but in a future pubrules upgrade - since process 2020 is only expected to impact CR+ docs, I would prefer we make as few changes as possible on pre-CR publication rules.

**Review by dontcallmedom — APPROVED** [review]

**Review by fantasai — APPROVED** [review]

File: p2020mockup/cr-6.2.8.2.html [hunk 1]
```diff
@@ -95,7 +95,7 @@ <h2>Status of This Document</h2>
     </p>
     (@@choose one of the 2 following paragraphs)
     <p>
-      This document may be updated, replaced or obsoleted at any time. It is inappropriate to
+      This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to
```
Comment (marcoscaceres) [inline p2020mockup/cr-6.2.8.2.html:98]: A little OCD, but Oxford comma?  
```suggestion
      This is a draft document and may be updated, replaced, or obsoleted by other documents at any time. It is inappropriate to
```

Comment (plehegar) [reply p2020mockup/cr-6.2.8.2.html:98]: we'll keep the comma for a future upgrade as well, since the goal here was to make the minimal amount of changes.

**Pull request merged**